bundles/binding/org.openhab.binding.bticino/README.md

```diff
@@ -120,13 +121,6 @@ OPEN=Heating ON
 20=Remote Control disabled
 21=Remote Control enabled
 
-22=At least one Probe OFF
-23=At least one Probe in protection
-24=At least one Probe in manual
-
-30=Failure discovered
-31=Central Unit battery KO
-
```

Comment:
Why are these values being removed here?

Reply:
It's a specific of OWN / Bticino. These Status Codes can be at the same time,
therefore, I summarised this status feedbacks in a aggregated String